File: Sistema de Control Clínico Veterinario/Sistema_Control_Clinico_Veterinario/logica/Veterinario.cs
namespace SistemaVeterinario.logica
{
    public class Veterinario : Persona
    {
        public string CodigoVet { get; set; }
        public string Especialidad { get; set; }
        public string Horario { get; set; }
        public string Consultorio { get; set; }

        public override string ToString() => $"Dr(a). {Nombre}";
    }
}

File: Sistema de Control Clínico Veterinario/bluej/Veterinario.java
public class Veterinario extends Persona {
    private String codigoVet;
    private String especialidad;
    private String horario;
    
    public Veterinario(String nombre, String dpi, String telefono, String email,
                       String codigo, String especialidad) {
        super(nombre, dpi, telefono, email);
        this.codigoVet = codigo;
        this.especialidad = especialidad;
    }
    
    public void atenderCita(Cita c) {
        c.setEstado("En atención");
    }
    
    public void registrarConsulta(Consulta c) {
        System.out.println("Consulta registrada para cita: " + c.getIdConsulta());
    }
    
    public String getEspecialidad() { return especialidad; }
}